src/test/java/seedu/address/storage/JsonAdaptedPersonTest.java

```diff
@@ -116,7 +117,7 @@ public void toModelType_invalidSubjects_throwsIllegalValueException() {
         invalidSubjects.add(new JsonAdaptedSubject(INVALID_SUBJECT));
         JsonAdaptedPerson person =
                 new JsonAdaptedPerson(VALID_NAME, VALID_PHONE, VALID_EMERGENCY_CONTACT,
-                        VALID_ADDRESS, "", invalidSubjects);
+                        VALID_ADDRESS, VALID_NOTE, invalidSubjects);
         assertThrows(IllegalValueException.class, person::toModelType);
     }
 
```

Comment:
Maybe can create another test toModelType_invalidNote_throwsIllegalValueException() here to test the line that codecov says isnt covered!